_docs_v5/4_guides/local_mode.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Overall very good.  Needs a section on how the directories are organized.

Reply:
Also would be good to talk about the syncing that occurs on startup by the openc3-init container.